diff --git a/dist/index.mjs b/dist/index.mjs
index 7ca992ae02b9c023cef4c49acf9dfe17bb06a621..f0a3c8a190a3ef13834505f369a2f08ce751bb7c 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -1996,14 +1996,7 @@ function verifyApiKey({ configurations, schema, deleteAllExpiredApiKeys }) {
 				},
 				key: null
 			});
-			return ctx.json({
-				valid: false,
-				error: {
-					message: API_KEY_ERROR_CODES.INVALID_API_KEY,
-					code: "INVALID_API_KEY"
-				},
-				key: null
-			});
+			throw error;
 		}
 		const { key: _, ...returningApiKey } = apiKey ?? {
 			key: 1,
